gears/flywheel/fsl-stats.json

```diff
@@ -0,0 +1,217 @@
+      "maximum": 100,
+      "id": "h"
+    },
+    "Windowed nbins for histogram": {
+      "description": "nbins, min, and max for output a histogram (for the thresholded/masked voxels only)\nwith nbins and histogram limits of min and max",
+      "optional": true,
+      "type": "string",
+      "id": "H",
+      "enum": [
+        ""
+      ]
+    },
+    "Robust min/max": {
+      "description": "output <robust min intensity> <robust max intensity>",
+      "id": "r",
+      "optional": true,
+      "type": "boolean"
+    },
+    "Min/max": {
+      "id": "R",
+      "description": "output <min intensity> <max intensity>",
+      "optional": true,
+      "type": "boolean"
+    },
+    "Entropy": {
+      "id": "e",
+      "description": "output mean entropy ; mean(-i*ln(i))",
+      "optional": true,
+      "type": "boolean"
+    },
+    "Entropy (nonzero)": {
+      "id": "E",
+      "description": "output mean entropy (of nonzero voxels)",
+      "optional": true,
+      "type": "boolean"
+    },
+    "Volume": {
+      "id": "v",
+      "description": "output <voxels> <volume>",
+      "optional": true,
+      "type": "boolean"
+    },
+    "Volume (nonzero)": {
+      "id": "V",
+      "description": "output <voxels> <volume> (for nonzero voxels)",
+      "optional": true,
+      "type": "boolean"
+    },
+    "ROI stats": {
+      "id": "w",
+      "description": "output smallest ROI <xmin> <xsize> <ymin> <ysize> <zmin> <zsize> <tmin> <tsize>\n containing nonzero voxels",
+      "optional": true,
+      "type": "boolean"
+    },
+    "Max voxel coords": {
+      "id": "x",
+      "description": "output co-ordinates of maximum voxel",
+      "optional": true,
+      "type": "boolean"
+    },
+    "Min voxel coords": {
+      "id": "X",
+      "description": "output co-ordinates of minimum voxel",
+      "optional": true,
+      "type": "boolean"
+    },
+    "Center of gravity (mm)": {
+      "id": "c",
+      "description": "output centre-of-gravity (cog) in mm coordinates",
+      "optional": true,
+      "type": "boolean"
+    },
+    "Center of gravity (voxels)": {
+      "id": "C",
+      "description": "output centre-of-gravity (cog) in voxel coordinates",
+      "optional": true,
+      "type": "boolean"
+    },
+    "Absolute values?": {
+      "id": "a",
+      "description": "use absolute values of all image intensities",
+      "optional": true,
+      "type": "boolean"
+    },
+    "NaN/Inf as zero?": {
+      "id": "n",
+      "description": "treat NaN or Inf as zero for subsequent stats",
+      "optional": true,
+      "type": "boolean"
+    }
+  },
+  "custom": {
+    "docker-image": "flywheel/fsl-stats:0.1.5_5.0",
+    "gear-builder": {
+      "category": "analysis",
+      "image": "flywheel/fsl-stats:0.1.5_5.0"
+    },
+    "flywheel": {
+      "suite": "FSL"
+    }
+  },
+  "description": "This gear returns statistical output for a given NIFTI image.",
+  "environment": {
+    "FLYWHEEL": "/flywheel/v0"
```

Comment:
I think some env var (e.g. PATH to the FSL bin) are missing here. Looking at the source code I am not getting how `fsl5.0-fslstats` is found from within the gear (given all env var defined in the docker image are removed)